pkg/install/v1alpha1/cstor_volume.go

```diff
@@ -621,7 +621,12 @@ spec:
     Calculate the replica count
     Add as many poolUid to resources as there is replica count
     */}}
-    {{- $poolUids := keys .ListItems.cvolPoolList.pools | randomize }}
+    {{- $replicaAffinityName:= .TaskResult.creategetpvc.replicaAntiAffinity -}}
```

Comment:
here the name is supposed to be `replicaAntiAffinityName` rather than 'replicaAffinityName`.. accordingly, need to update the 'if' condition

Reply:
done